### ISSUE TITLE
Trying to start the sample Java app through the default setup leads to ECS Task startup failure

### ISSUE THREAD

**Issue by cfregly** [awslabs/aws-saas-boost#182]
Copied error logs below (pulled from ECS CloudWatch Logs)

```
2022-01-12T14:19:47.093-08:00	2022-01-12 22:19:47.093 WARN 7 --- [ main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.context.AppliationContextException: Unable to start web server; nested exception is org.springframework.boot.web.server.WebServerException: Unable to start embedded Tomcat

2022-01-12T14:19:47.166-08:00	2022-01-12 22:19:47.166 INFO 7 --- [ main] ConditionEvaluationReportLoggingListener :

2022-01-12T14:19:47.166-08:00	Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.

2022-01-12T14:19:47.257-08:00	2022-01-12 22:19:47.257 ERROR 7 --- [ main] o.s.boot.SpringApplication : Application run failed

2022-01-12T14:19:47.257-08:00	org.springframework.context.ApplicationContextException: Unable to start web server; nested exception is org.springframework.boot.web.server.WebServerException: Unable to start embedded Tomcat

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.onRefresh(ServletWebServerApplicationContext.java:163) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:577) ~[spring-context-5.3.13.jar!/:5.3.13]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:145) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:730) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:412) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.SpringApplication.run(SpringApplication.java:302) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:164) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	at com.amazon.aws.partners.saasfactory.HelloWorld.main(HelloWorld.java:30) ~[classes!/:na]

2022-01-12T14:19:47.257-08:00	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]

2022-01-12T14:19:47.257-08:00	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]

2022-01-12T14:19:47.257-08:00	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]

2022-01-12T14:19:47.257-08:00	at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49) ~[app.war:na]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.loader.Launcher.launch(Launcher.java:108) ~[app.war:na]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58) ~[app.war:na]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.loader.WarLauncher.main(WarLauncher.java:59) ~[app.war:na]

2022-01-12T14:19:47.257-08:00	Caused by: org.springframework.boot.web.server.WebServerException: Unable to start embedded Tomcat

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:142) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.<init>(TomcatWebServer.java:104) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getTomcatWebServer(TomcatServletWebServerFactory.java:473) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getWebServer(TomcatServletWebServerFactory.java:206) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.createWebServer(ServletWebServerApplicationContext.java:182) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.onRefresh(ServletWebServerApplicationContext.java:160) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	... 15 common frames omitted

2022-01-12T14:19:47.257-08:00	Caused by: org.apache.catalina.LifecycleException: A child container failed during start

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:938) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:263) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.core.StandardService.startInternal(StandardService.java:432) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:927) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.startup.Tomcat.start(Tomcat.java:486) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:123) ~[spring-boot-2.6.1.jar!/:2.6.1]

2022-01-12T14:19:47.257-08:00	... 20 common frames omitted

2022-01-12T14:19:47.257-08:00	Caused by: java.util.concurrent.ExecutionException: org.apache.catalina.LifecycleException: A child container failed during start

2022-01-12T14:19:47.257-08:00	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[na:na]

2022-01-12T14:19:47.257-08:00	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[na:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:926) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	... 28 common frames omitted

2022-01-12T14:19:47.257-08:00	Caused by: org.apache.catalina.LifecycleException: A child container failed during start

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:938) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:835) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1396) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1386) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]

2022-01-12T14:19:47.257-08:00	at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:140) ~[na:na]

2022-01-12T14:19:47.257-08:00	at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:919) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.257-08:00	... 28 common frames omitted

2022-01-12T14:19:47.257-08:00	Caused by: java.util.concurrent.ExecutionException: org.apache.catalina.LifecycleException: Failed to start component [StandardEngine[Tomcat].StandardHost[localhost].TomcatEmbeddedContext[]]

2022-01-12T14:19:47.257-08:00	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[na:na]

2022-01-12T14:19:47.257-08:00	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[na:na]

2022-01-12T14:19:47.258-08:00	at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:926) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.258-08:00	... 36 common frames omitted

2022-01-12T14:19:47.258-08:00	Caused by: org.apache.catalina.LifecycleException: Failed to start component [StandardEngine[Tomcat].StandardHost[localhost].TomcatEmbeddedContext[]]

2022-01-12T14:19:47.258-08:00	at org.apache.catalina.util.LifecycleBase.handleSubClassException(LifecycleBase.java:440) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.258-08:00	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:198) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.258-08:00	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1396) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.258-08:00	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1386) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.258-08:00	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]

2022-01-12T14:19:47.258-08:00	at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.258-08:00	at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:140) ~[na:na]

2022-01-12T14:19:47.258-08:00	at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:919) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.258-08:00	... 36 common frames omitted

2022-01-12T14:19:47.258-08:00	Caused by: java.lang.NoClassDefFoundError: org/apache/tomcat/util/descriptor/tld/TldParser

2022-01-12T14:19:47.258-08:00	at org.apache.jasper.servlet.TldScanner.<init>(TldScanner.java:84) ~[tomcat-embed-jasper-9.0.35.jar!/:9.0.35]

2022-01-12T14:19:47.258-08:00	at org.apache.jasper.servlet.JasperInitializer.newTldScanner(JasperInitializer.java:100) ~[tomcat-embed-jasper-9.0.35.jar!/:9.0.35]

2022-01-12T14:19:47.258-08:00	at org.apache.jasper.servlet.JasperInitializer.onStartup(JasperInitializer.java:81) ~[tomcat-embed-jasper-9.0.35.jar!/:9.0.35]

2022-01-12T14:19:47.258-08:00	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5219) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.258-08:00	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.55.jar!/:na]

2022-01-12T14:19:47.258-08:00	... 42 common frames omitted

2022-01-12T14:19:47.258-08:00	Caused by: java.lang.ClassNotFoundException: org.apache.tomcat.util.descriptor.tld.TldParser

2022-01-12T14:19:47.258-08:00	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:476) ~[na:na]

2022-01-12T14:19:47.258-08:00	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589) ~[na:na]

2022-01-12T14:19:47.258-08:00	at org.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:151) ~[app.war:na]

2022-01-12T14:19:47.258-08:00	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522) ~[na:na]

2022-01-12T14:19:47.258-08:00	... 47 common frames omitted
```

**Comment by cfregly** [comment]
solution is to temporarily revert to an older version of pom.xml with a diff version of springboot:

```diff --git a/samples/java/pom.xml b/samples/java/pom.xml
index b4c53ce..286c123 100644
--- a/samples/java/pom.xml
+++ b/samples/java/pom.xml
@@ -64,18 +64,18 @@ limitations under the License.
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
-            <version>2.6.1</version>
+            <version>2.3.0.RELEASE</version>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-tomcat</artifactId>
-            <version>2.6.1</version>
+            <version>2.3.0.RELEASE</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-jdbc</artifactId>
-            <version>2.6.1</version>
+            <version>2.3.0.RELEASE</version>
         </dependency>
         <dependency>
             <groupId>org.apache.logging.log4j</groupId>
```

**Comment by cfregly** [comment]
after applying the fix above, i am now seeing this;

```
2022-01-12 22:46:12.927  ERROR SaaSBoostInstall - java.lang.RuntimeException: Error with CloudFormation stack sb-prod. Check the events in the AWS CloudFormation Console
        at com.amazon.aws.partners.saasfactory.saasboost.SaaSBoostInstall.updateCloudFormationStack(SaaSBoostInstall.java:1527)
        at com.amazon.aws.partners.saasfactory.saasboost.SaaSBoostInstall.updateMetricsStack(SaaSBoostInstall.java:797)
        at com.amazon.aws.partners.saasfactory.saasboost.SaaSBoostInstall.updateSaaSBoost(SaaSBoostInstall.java:409)
        at com.amazon.aws.partners.saasfactory.saasboost.SaaSBoostInstall.start(SaaSBoostInstall.java:203)
        at com.amazon.aws.partners.saasfactory.saasboost.SaaSBoostInstall.main(SaaSBoostInstall.java:150)
```

**Comment by cfregly** [comment]
more Tomcat issues:  

```
2022-01-12T15:07:47.629-08:00 | at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:919) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.629-08:00 | ... 36 common frames omitted
  | 2022-01-12T15:07:47.629-08:00 | Caused by: java.lang.NoClassDefFoundError: org/apache/tomcat/util/descriptor/tld/TldParser
  | 2022-01-12T15:07:47.629-08:00 | at org.apache.jasper.servlet.TldScanner.<init>(TldScanner.java:84) ~[tomcat-embed-jasper-9.0.35.jar!/:9.0.35]
  | 2022-01-12T15:07:47.629-08:00 | at org.apache.jasper.servlet.JasperInitializer.newTldScanner(JasperInitializer.java:100) ~[tomcat-embed-jasper-9.0.35.jar!/:9.0.35]
  | 2022-01-12T15:07:47.629-08:00 | at org.apache.jasper.servlet.JasperInitializer.onStartup(JasperInitializer.java:81) ~[tomcat-embed-jasper-9.0.35.jar!/:9.0.35]
  | 2022-01-12T15:07:47.629-08:00 | at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5219) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.629-08:00 | at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.629-08:00 | ... 42 common frames omitted
  | 2022-01-12T15:07:47.629-08:00 | Caused by: java.lang.ClassNotFoundException: org.apache.tomcat.util.descriptor.tld.TldParser
  | 2022-01-12T15:07:47.629-08:00 | at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:476) ~[na:na]
  | 2022-01-12T15:07:47.629-08:00 | at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589) ~[na:na]
  | 2022-01-12T15:07:47.629-08:00 | at org.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:151) ~[app.war:na]
  | 2022-01-12T15:07:47.629-08:00 | at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522) ~[na:na]
  | 2022-01-12T15:07:47.629-08:00 | ... 47 common frames omitted
  | 2022-01-12T15:07:47.630-08:00 | 2022-01-12 23:07:47.630 INFO 7 --- [ main] o.apache.catalina.core.StandardService : Stopping service [Tomcat]
  | 2022-01-12T15:07:47.739-08:00 | 2022-01-12 23:07:47.739 WARN 7 --- [ main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.context.ApplicationContextException: Unable to start web server; nested exception is org.springframework.boot.web.server.WebServerException: Unable to start embedded Tomcat
  | 2022-01-12T15:07:47.807-08:00 | 2022-01-12 23:07:47.807 INFO 7 --- [ main] ConditionEvaluationReportLoggingListener :
  | 2022-01-12T15:07:47.807-08:00 | Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
  | 2022-01-12T15:07:47.840-08:00 | 2022-01-12 23:07:47.839 ERROR 7 --- [ main] o.s.boot.SpringApplication : Application run failed
  | 2022-01-12T15:07:47.840-08:00 | org.springframework.context.ApplicationContextException: Unable to start web server; nested exception is org.springframework.boot.web.server.WebServerException: Unable to start embedded Tomcat
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.onRefresh(ServletWebServerApplicationContext.java:163) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:577) ~[spring-context-5.3.13.jar!/:5.3.13]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:145) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:730) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:412) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.SpringApplication.run(SpringApplication.java:302) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:164) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | at com.amazon.aws.partners.saasfactory.HelloWorld.main(HelloWorld.java:30) ~[classes!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49) ~[app.war:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.loader.Launcher.launch(Launcher.java:108) ~[app.war:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.loader.Launcher.launch(Launcher.java:58) ~[app.war:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.loader.WarLauncher.main(WarLauncher.java:59) ~[app.war:na]
  | 2022-01-12T15:07:47.840-08:00 | Caused by: org.springframework.boot.web.server.WebServerException: Unable to start embedded Tomcat
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:142) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.<init>(TomcatWebServer.java:104) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getTomcatWebServer(TomcatServletWebServerFactory.java:473) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getWebServer(TomcatServletWebServerFactory.java:206) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.createWebServer(ServletWebServerApplicationContext.java:182) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.onRefresh(ServletWebServerApplicationContext.java:160) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | ... 15 common frames omitted
  | 2022-01-12T15:07:47.840-08:00 | Caused by: org.apache.catalina.LifecycleException: A child container failed during start
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:938) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:263) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.StandardService.startInternal(StandardService.java:432) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:927) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.startup.Tomcat.start(Tomcat.java:486) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:123) ~[spring-boot-2.6.1.jar!/:2.6.1]
  | 2022-01-12T15:07:47.840-08:00 | ... 20 common frames omitted
  | 2022-01-12T15:07:47.840-08:00 | Caused by: java.util.concurrent.ExecutionException: org.apache.catalina.LifecycleException: A child container failed during start
  | 2022-01-12T15:07:47.840-08:00 | at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:926) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | ... 28 common frames omitted
  | 2022-01-12T15:07:47.840-08:00 | Caused by: org.apache.catalina.LifecycleException: A child container failed during start
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:938) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:835) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1396) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1386) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:140) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:919) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | ... 28 common frames omitted
  | 2022-01-12T15:07:47.840-08:00 | Caused by: java.util.concurrent.ExecutionException: org.apache.catalina.LifecycleException: Failed to start component [StandardEngine[Tomcat].StandardHost[localhost].TomcatEmbeddedContext[]]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:926) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | ... 36 common frames omitted
  | 2022-01-12T15:07:47.840-08:00 | Caused by: org.apache.catalina.LifecycleException: Failed to start component [StandardEngine[Tomcat].StandardHost[localhost].TomcatEmbeddedContext[]]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.util.LifecycleBase.handleSubClassException(LifecycleBase.java:440) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:198) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1396) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1386) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:140) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:919) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | ... 36 common frames omitted
  | 2022-01-12T15:07:47.840-08:00 | Caused by: java.lang.NoClassDefFoundError: org/apache/tomcat/util/descriptor/tld/TldParser
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.jasper.servlet.TldScanner.<init>(TldScanner.java:84) ~[tomcat-embed-jasper-9.0.35.jar!/:9.0.35]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.jasper.servlet.JasperInitializer.newTldScanner(JasperInitializer.java:100) ~[tomcat-embed-jasper-9.0.35.jar!/:9.0.35]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.jasper.servlet.JasperInitializer.onStartup(JasperInitializer.java:81) ~[tomcat-embed-jasper-9.0.35.jar!/:9.0.35]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5219) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.55.jar!/:na]
  | 2022-01-12T15:07:47.840-08:00 | ... 42 common frames omitted
  | 2022-01-12T15:07:47.840-08:00 | Caused by: java.lang.ClassNotFoundException: org.apache.tomcat.util.descriptor.tld.TldParser
  | 2022-01-12T15:07:47.840-08:00 | at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:476) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00 | at org.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:151) ~[app.war:na]
  | 2022-01-12T15:07:47.840-08:00 | at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522) ~[na:na]
  | 2022-01-12T15:07:47.840-08:00

<br class="Apple-interchange-newline">
```

**Comment by brtrvn** [comment]
Closed with PR #183
